src/platform/plugins/shared/dashboard/public/dashboard_container/component/grid/_dashboard_panel.scss

```diff
@@ -1,7 +1,7 @@
 // LAYOUT MODES
 // Adjust borders/etc... for non-spaced out and expanded panels
 .dshLayout-withoutMargins {
-  margin-top: $euiSizeS;
+  padding-top: $euiSizeS;
```

Comment:
Switching to padding so that expanded panels don't extend past the height of the viewport

Reply:
Amazing attention to detail